core/modules/dashboard/dashboard.module

```diff
@@ -222,7 +222,7 @@ function dashboard_preprocess_layout(&$variables) {
 function dashboard_preprocess_block(&$variables) {
   /* @var Layout $layout */
   $layout = $variables['layout'];
-  if ($layout->hasContexts(array('dashboard'))) {
+  if (is_a($layout, 'Layout') && $layout->hasContexts(array('dashboard'))) {
```

Comment:
@olafgrabienski I'm thinking we should have the same thing in the function just above this one. Can you make it:

```
function dashboard_preprocess_layout(&$variables) {
  /* @var Layout $layout */
  $layout = $variables['layout'];
  if (is_a($layout, 'Layout') && $layout->hasContexts(array('dashboard'))) {
```

Reply:
Sure, done!